docs/greenboot.md

```diff
@@ -79,3 +94,19 @@ boot_success=1
 > core MicroShift services. Users should install their own workload validation
 > scripts using `greenboot` facilities to ensure the successful operation after
 > system upgrades.
+
+## The `systemd` Journal Service Configuration
+
+The default configuration of the `systemd` journal service stores the data in
+the volatile `/run/log/journal` directory, which does not persist after a system
+boot. To monitor `greenboot` activities across system boots, it is recommended to
+enable the journal data persistency by creating the `/var/log/journal` directory
+and setting limits on the maximal journal data size.
+
+Run the following commands to configure the journal data persistency and limits.
```

Comment:
It's also possible to include the settings in a new file in /etc/systemd/journald.conf.d/ so I wonder if that's a better approach for someone building their own images.

Reply:
I have included this configuration in kickstart. I'm not sure we would want to impose system logging configuration from MicroShift RPM.

Reply:
I just meant in our docs, not in our RPMs.